system_tests/seqfeed_test.go

```diff
@@ -491,3 +495,102 @@ func TestPopulateFeedBacklog(t *testing.T) {
 		t.Fatal("BlockHashMismatchLogMsg was logged unexpectedly")
 	}
 }
+
+func TestRegressionInPopulateFeedBacklog(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	builder := NewNodeBuilder(ctx).DefaultConfig(t, true)
+	builder.BuildL1(t)
+
+	userAccount := "User2"
+	builder.L2Info.GenerateAccount(userAccount)
+
+	// Guarantees that nodes will rely only on the feed to receive messages
+	builder.nodeConfig.BatchPoster.Enable = false
```

Comment:
as we recently discovered, this will turn batch poster everywhere and should be adapted to #3829; bringing @ganeshvanahalli attention in case this PR will get merged first